clients/py/dev-requirements.txt

```diff
@@ -1,4 +1,6 @@
 build
+# https://stackoverflow.com/questions/73929564/entrypoints-object-has-no-attribute-get-digital-ocean
```

Comment:
How this is related?

Reply:
It's not. It's needed to unbreak development CI.